regression/filters/magnitude_det_FIR/filter_03/input.c

```diff
@@ -0,0 +1,29 @@
+//fhp10Equi
```

Comment:
Sorry, I can't delete these comments at the beginning of each filter in the regressions folder. They are there to identify which filter was used. I'd better delete the entire filters regression folder, if that's the case.